modules/command/src/main/java/org/apache/fluo/command/FluoGetJars.java

```diff
@@ -33,40 +36,83 @@
 
   private static final Logger log = LoggerFactory.getLogger(FluoGetJars.class);
 
+  public static class GetJarsOpts {
```

Comment:
It seems like this, and many other classes, could extend another an abstract class that provides the `-a` and `-D` options along with the associated methods.

Reply:
This is a good idea.  I will work on this.